### PR TITLE
feat: add eslint

### PR DESCRIPTION
**Pull request by troZee**
# Summary
Add ESlint to project

### CODE REVIEW

**Comment by dratwas** [comment]
Hey, @troZee I had to remove `resource_class` since we have a free plan. PR looks good but please add a job to CI where you run lint check :)

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "flow": "flow"
+    "flow": "flow",
+    "lint": "eslint .",
+    "fix": "eslint . --fix --ext .js"
```
Comment (ferrannp) [inline package.json]: I find this redundant. I'd say `yarn lint --fix` is enough. Or at least the script to be `lint:fix`. Just `fix` is very generic.

Comment (dratwas) [reply package.json]: Yeah I would leave the script for fixing but as @ferrannp said `fix` is too generic. Let's use `lint:fix`

**Comment by ferrannp** [comment]
Thank you!

**Pull request merged**